promql/promqltest/testdata/functions.test

```diff
@@ -494,6 +494,7 @@ load 5m
 	test_sgn{src="sgn-d"}	-50
 	test_sgn{src="sgn-e"}	0
 	test_sgn{src="sgn-f"}	100
+    test_sgn{src="sgn-histogram"} {{schema:1 sum:1 count:1}}
```

Comment:
This line uses spaces, but the previous lines use tabs. Depending on the tab width, that might lead to weird alignment. (E.g. GH displays this file with 8-space tabs, so the lines above are too far indented.)

I recommend to use a tab in this line too. (Or change all lines to use spaces.)

Reply:
Thanks. I'll fix this.